website/docs/terms/etl.md

```diff
@@ -0,0 +1,119 @@
+
+### Load
+
+In the final stage, the transformed data is loaded into your target data warehouse. Once this transformed data is in its final destination, it’s most commonly exposed to end business users either in a BI tool or in the data warehouse directly.
+
+The ETL workflow implies that your raw data does not live in your data warehouyse. *Because transformations occur before load, only transformed data lives in your data warehouse in the ETL process.* This can make it harder to ensure that transformations are performing the correct functionality.
+
+## ETL vs ELT
+
+You may read other articles or technical documents that use ETL and ELT interchangeably. On paper, the only difference is the order in which the T and the L appear. However, this mere switching of letters dramatically changes the way data exists in and flows through a business’ system.
+
+In both processes, data from different data sources is extracted in similar ways. However, in ELT, data is then directly loaded into the target data platform versus being transformed in ETL. Now, via ELT workflows, both raw and transformed data can live in a data warehouse. In ELT workflows, data folks have the flexibility to model the data after they’ve had the opportunity to explore and analyze the raw data. ETL workflows can be more constraining since transformations happen immediately after extraction. We break down some of the other major differences between the two below:
+
+|  | ETL | ELT |
+|---|---|---|
+| Programming skills required | Often requires custom scripts or considerable data engineering lift to extract and transform data prior to load. | Often requires little to no code to extract and load data into your data warehouse. |
+| Paritioning of layers | Extraction, load, and transformation layers can be explicitly separated out by different products. | ETL processes are often encapsulated in one product. |
+| Flexibility in transformations | Since transformations take place last, there is greater flexibility in the modeling process. Worry first about getting your data in one place, then you have time to explore the data to understand the best way to transform it. | Because transformation occurs before data is loaded into the target location, teams must conduct thorough work prior to make sure data is transformed properly. |
+| [Data team roles](https://www.getdbt.com/data-teams/analytics-job-descriptions/) | ELT workflows empower data team members who know SQL to create their own extraction and loading pipelines and transformations. | ETL workflows often require teams with greater technical skill to create and maintain pipelines. |
+
+While ELT is growing in adoption, it’s still important to talk about when ETL might be appropriate and where you'll see challenges with the ETL workflow.
+
+## ETL use cases and challenges
+
+### Example ETL use cases
```

Comment:
@johnblust FYI this looks like an old version of the ETL post

Reply:
Great catch!!! Huge fumble on my part. Fixed this in the latest commit.